src/olympia/stats/management/commands/index_stats.py

```diff
@@ -76,24 +81,20 @@ def index_stats(index, addons=None, dates=None):
                 continue
 
             num_days = (limits['max'] - limits['min']).days
-            today = date.today()
```

Comment:
This command was "broken" in 3 different ways:
- Using `today` instead of the `max` date as the starting point
- Overlapping the `date_range`s in each iteration by one day
- Triggering tasks even if there was no data for that range 

None of these points terribly matter in production, where we're pretty much guaranteed to have data for every day until today, but in tests and locally it was spawning hundreds of useless empty tasks.